docker-compose.example.yml

```diff
@@ -0,0 +1,68 @@
+version: "3"
+services:
+  # The postgres server
```

Comment:
In case you want to containerize `rabbitmq`, adding [this](https://gist.github.com/rafaelgalani/6730793c77b6f9bd6cad5bd30d2e7d2c) in `services` section should work.